core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -381,6 +384,11 @@ public boolean getHostingRequested() {
     return onDemandHostingRequested;
   }
 
+  public UnSplittableMetadata getUnSplittable() {
```

Comment:
This comment is not related to this line.  Need to update the toString() method on this class.